[CSSProperties.json] Remove statuses for unimplemented properties

nt1m opened the pull request:
#### dd51d03f3c50d6a4c237a5a7190029a1a8055b1e
<pre>
[CSSProperties.json] Remove statuses for unimplemented properties
<a href="https://bugs.webkit.org/show_bug.cgi?id=257925">https://bugs.webkit.org/show_bug.cgi?id=257925</a>
rdar://110555626

Reviewed by Simon Fraser.

- &quot;Under Consideration&quot; / &quot;Not Considering&quot; belong in: <a href="https://github.com/WebKit/standards-positions">https://github.com/WebKit/standards-positions</a>
- &quot;Not Implemented&quot; / &quot;Removed&quot; data belongs in sources like Can I Use? or BCD, which we should try to keep up to date.

* Source/WebCore/css/CSSProperties.json:
* Websites/webkit.org/wp-content/themes/webkit/css-status.php:

Canonical link: <a href="https://commits.webkit.org/265049@main">https://commits.webkit.org/265049@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ca4f975687d484d6db8a739283ace590e70dd239

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/9630 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/9885 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/10130 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/11291 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/9411 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/9640 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/11865 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/9841 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/12333 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/9782 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/10632 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/8195 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/11450 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/7928 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/8752 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/16161 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/9022 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/8901 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/12238 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/9392 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/7627 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/8581 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/12802 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/1105 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/9166 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

smfr commented on Websites/webkit.org/wp-content/themes/webkit/css-status.php:
```diff
@@ -723,7 +677,7 @@ function xhrPromise(url) {
                     <ul class="properties" id="properties-container"></ul>
 
-                    <p>Cannot find something? You can contact <a href="https://twitter.com/webkit">@webkit</a> on Twitter or contact the <a href="https://lists.webkit.org/mailman/listinfo/webkit-help">webkit-help</a> mailing list for questions.</p>
+                    <p>Cannot find something? Standards positions can be found on <a href="https://github.com/WebKit/standards-positions">this Github repository</a>. You can also contact <a href="https://twitter.com/webkit">@webkit</a> on Twitter or contact the <a href="https://lists.webkit.org/mailman/listinfo/webkit-help">webkit-help</a> mailing list for questions.</p>
```
I don't think we should encourage use of Twitter any more.

smfr commented on Source/WebCore/css/CSSProperties.json:
```diff
@@ -9459,51 +9210,6 @@
             },
             "status": "non-standard"
-        },
-        "fill-color": {
```
Let's file a supportive position on the Fill and Stroke module.

nt1m replied:
https://github.com/WebKit/standards-positions/issues/202

nt1m committed:
[CSSProperties.json] Remove statuses for unimplemented properties
https://bugs.webkit.org/show_bug.cgi?id=257925
rdar://110555626

Reviewed by Simon Fraser.

- "Under Consideration" / "Not Considering" belong in: https://github.com/WebKit/standards-positions
- "Not Implemented" / "Removed" data belongs in sources like Can I Use? or BCD, which we should try to keep up to date.

* Source/WebCore/css/CSSProperties.json:
* Websites/webkit.org/wp-content/themes/webkit/css-status.php:

Canonical link: https://commits.webkit.org/265049@main
